.github/workflows/tests.yml

```diff
@@ -3,6 +3,9 @@ name: Test
 on:
   workflow_dispatch:
   push:
+    branches:
+      - main
```

Comment:
We currently don't have a `main` branch, so this is redundant. If we some day decide to rename `master` to `main`, we'll have to grep for `master` anyway.